seatunnel-connectors-v2/connector-redis/src/main/java/org/apache/seatunnel/connectors/seatunnel/redis/config/RedisConfig.java

```diff
@@ -127,6 +127,25 @@ public enum HashKeyParseMode {
                             "batch_size is used to control the size of a batch of data during read and write operations"
                                     + ",default 10");
 
+    public static final Option<String> VALUE_FIELD =
+            Options.key("value_field")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "The field of value you want to write to redis, support string list set zset");
```

Comment:
would it be better to add end-to-end tests for all types, including string, list, set, and zset?